geonode/qgis_server/tests/test_views.py

```diff
@@ -564,30 +564,29 @@ def test_add_delete_style(self):
             set(expected_list_style),
             set([style.name for style in actual_list_style]))
 
-        # Check new default
+        # Check new default, namely: new_style
         default_style_url = reverse(
             'qgis_server:default-qml',
             kwargs={
-                'layername': layer.name})
+                'layername': layer.name,
+                'style_name': 'new_style'})
 
-        response = self.client.get(default_style_url)
+        response = self.client.post(default_style_url)
```

Comment:
Why you post? The test below is to test default style query (using get). Maybe you want to create another section tests?

Reply:
yes, i have created a new test section for the post request. 
the post sets a new style.